projects/cashmere/src/lib/button/button.component.ts

```diff
@@ -4,13 +4,7 @@
 import {ChangeDetectionStrategy, Component, ElementRef, HostBinding, Input, Renderer2, ViewEncapsulation} from '@angular/core';
 import {parseBooleanAttribute} from '../util';
 
-const supportedColors = ['primary', 'primary-alt', 'destructive', 'neutral', 'secondary', 'tertiary'];
-
-export function validateButtonColor(btnColor: string) {
-    if (supportedColors.indexOf(btnColor) < 0) {
-        throw Error('Unsupported color input value: ' + btnColor);
-    }
-}
+export type ButtonStyle = 'primary' | 'primary-alt' | 'destructive' | 'neutral' | 'secondary' | 'link' | 'link-inline';
```

Comment:
We're going to need to keep the old stuff and update the list of valid styles. This is also causing a test to fail. The buttonStyle is fine and the reason that was removed was that dgeni couldn't generate docs for it